WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -372,6 +372,9 @@ private void endMigration() {
 
     public static void registerMigrationListener(MigrationListener listener) {
         sMigrationListener = listener;
+        if (sIsMigrationError) {
+            sMigrationListener.onError();
```

Comment:
Yeah, I'm not a fan of null checking everything, I prefer to crash in this case. Aat least we'll now someone made a mistake by setting a null listener ;)